### PR TITLE
Python Build Fix

### PR DESCRIPTION
**Pull request by shubham1206agra**
Changed the strict hard-coding to softer one using glob library.

### CODE REVIEW

**Comment by shubham1206agra** [comment]
Ping: @zoq @rcurtin 
Please review it.

**Review by zoq — APPROVED** [review]
Thanks for looking into this.

File: src/mlpack/bindings/python/copy_artifacts.py [hunk 1]
```diff
@@ -6,18 +6,14 @@
...
+import glob
 
-directory = 'build/lib.' + \
-            sysconfig.get_platform() + \
-            '-' + \
-            str(sys.version_info[0]) + \
-            '.' + \
-            str(sys.version_info[1]) + \
-            '/mlpack/'
+# New directory structure
```
Comment (rcurtin) [inline src/mlpack/bindings/python/copy_artifacts.py:14]:
```suggestion
# Match any lib.$platform*/mlpack/ directory.
```
This might be a cleaner comment.

**Review by rcurtin — APPROVED** [review]
Awesome, thank you for looking into this!  I think when I originally wrote this file, I was worried about getting the wrong version, but I suspect that that case will never actually be encountered in the wild.  I suppose you would have to configure a build directory, build the Python bindings, then *change* the Python version in CMake *without* cleaning the build directory and then rebuild.  I doubt that anyone will ever actually do that...

**Comment by rcurtin** [comment]
Thanks again for the contribution! :rocket:

**Pull request merged**